### ISSUE TITLE
setFullScreen(true) will crash jre on macos with M1 chip

### ISSUE THREAD

**Issue by kunterbunt2** [libgdx/libgdx#6896]
Please ensure you have given all the following requested information in your report.

#### Issue details
To simplify the debugging I created a simple test using the project creator wizard.
https://github.com/kunterbunt2/libgdx4m1
if run with full screen mode, it will crash the jre on a mac mini with m1 chip.
crash report at https://github.com/kunterbunt2/libgdx4m1/blob/main/fullScreenCrash.txt

#### Reproduction steps/code
import project at https://github.com/kunterbunt2/libgdx4m1 as gradle project into Eclipse with jre 1.8.
execute desktop/src/com/mygdx/game/full/screen)/FullScreen.java

#### Version of libGDX and/or relevant dependencies
libgdx 1.11.0 + gdx-lwjgl3-glfw-awt-macos

#### Stacktrace
https://github.com/kunterbunt2/libgdx4m1/blob/main/fullScreenCrash.txt

#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [ ] Windows
- [ ] Linux
- [X] macOS

**Comment by crykn** [comment]
Does the issues also happen without the glfw-awt-macos extension (i.e. with the [`-XstartOnFirstThread` argument](https://libgdx.com/news/2021/07/devlog-7-lwjgl3#do-i-need-to-do-anything-else))?

**Comment by kunterbunt2** [comment]
I will check that soon as I can, but I think I tried that before creating the ticket.

**Comment by crykn** [comment]
Regarding number 3: did you use the -XStartOnFirtsThread argument and the glfw-awt-macos extension at the same time? Then a crash is expected, they are exclusive to one another.

**Comment by kunterbunt2** [comment]
No, I did 1, then 2, then 3. so dependency was no longer there.

**Comment by kunterbunt2** [comment]
ok @crykn , I retested and checked eclipse external dependency and as you suspected, the macos awt library was still there. I then refreshed the gradle project and rechecked and it was gone. Something I usually always do after editing the build.gradle file, but must have missed to do this time.
Now I can confirm that if I remove the glfw-awt-macos extension and add the -XstartOnFirstThread argument full screen mode will work and not crash.
This means that we have a workaround!
Thanks!
